packages/react-tabs/Spec.md

```diff
@@ -63,211 +72,270 @@ The existing components are:
+</TabList>
+
+<TabList appearance="transparent">
+  {/* ... */}
 </TabList>
 ```
 
--or-
+Subtle
+
+The subtle appearance is similar to transparent, but a light background is displayed when hovering over a tab.
 
 ```tsx
-<TabList vertical>
-  <Tab vertical>One</Tab>
-  <Tab vertical>Two</Tab>
-  <Tab vertical>Three</Tab>
-</TabList>
+<TabList appearance="subtle">{/* ... */}</TabList>
 ```
 
-### Appearance
+### Size
 
-> TODO: Confirm with design if transparent is the default.
-> TODO: Confirm with design there is not a manilla folder tab appearance.
-> TODO: Ask design what the subtle and transparent styles are for focus, hover, pressed, disabled.
+Small and medium sizes are supported.
+The size affects the padding and spacing between elements.
+Size can also affect the default icon (and badge) size.
+The default size is medium.
 
 ```tsx
-<TabList>{/* ... */}</TabList>
+<TabList size="small">
+  {/* ... */}
+</TabList>
 
-<TabList appearance="transparent">{/* ... */}</TabList>
+<TabList size="medium">
+  {/* ... */}
+</TabList>
 ```
 
+### With Icon
+
+When the icon slot is filled, an icon is positioned before the tab content.
+
 ```tsx
-<TabList appearance="subtle">{/* ... */}</TabList>
+<TabList>
+  <Tab icon={<CheckboxComposite />} value="allowed">
+    Allowed
+  </Tab>
+  <Tab icon={<BlockedSite />} value="blocked">
+    Blocked
+  </Tab>
+</TabList>
 ```
 
-### Badges
+An icon only tab is displayed when the icon slot is filled and tab content is omitted.
+
+```tsx
+<TabList>
+  <Tab icon={<CheckboxComposite />} value="allowed" />
+  <Tab icon={<BlockedSite />} value="blocked" />
+</TabList>
+```
 
-> TODO: How to support multiple badge types as a slot?
-> TODO: How to specify icon for basic badge in a slot?
+### With Badge
+
+Adding a badge to a tab is done through the BadgeTab variant.
+This follows a extensible approach of MenuItem rather than defining a badge slot on the Tab component.
+The BadgeTab has a badge slot (BadgeProps) and leverages hooks from Badge.
+
+The badge is positioned relative to the icon depending on the verticalTabContent and size props.
+The BadgeTab variant support content and icon in addition to the badge.
 
 ```tsx
 <TabList>
-  <Tab>Files</Tab>
-  <Tab>Search</Tab>
-  <Tab badge={<CounterBadge count={5} />}>Messages</Tab>
+  <BadgeTab badge="24">Inbox</BadgeTab>
+  <BadgeTab badge="24" icon={<MailMessage />}>
+    Inbox
+  </BadgeTab>
 </TabList>
 ```
 
-### Overflow
+### TabPanel (future)
+
+The initial vNext package will **not** provide a tab panel, but the tab list will be designed to support a tab panel in the future through composition.
+
+Why not now?
 
-Handling overflow is a built-in feature of tabs and replaceable via a slot.
+- There are many approaches to handling tab panel content: show/hide, conditional render, navigation through a router
+- We're not providing a manilla folder tab appearance, so there is no need for the tab panel to support styling for when the components are co-located.
```

Comment:
What do we mean by `manilla folder tab appearance` in this case?

Reply:
When there is a rounded border on the left, top, and right in the horizontal case and on the top. left, and bottom in the vertical case.

Reply:
Wasn't aware of that expression, thanks for clarifying!